policyengine_us/variables/household/income/person/overtime_income.py

```diff
@@ -9,3 +9,22 @@ class overtime_income(Variable):
     definition_period = YEAR
 
     # This variable only exists for the purpose of the tax_exempt_reform
+
+    def formula(person, period, parameters):
+        p = parameters(period).gov.irs.income.exemption.overtime
+        worked_hours = person("weekly_hours_worked", period)
+        weekly_pay = person("employment_income", period) / WEEKS_IN_YEAR
+        eligible = person("is_eligible_for_fsla_overtime_protection", period)
+
+        non_overtime_hourly_rate = weekly_pay / (
+            p.hours_threshold
+            + max_((worked_hours - p.hours_threshold), 0) * p.rate_multiplier
+        )
+        overtime_income = (
+            max_((worked_hours - p.hours_threshold), 0)
+            * non_overtime_hourly_rate
+            * p.rate_multiplier
+        )
+
+        if eligible:
```

Comment:
remove this - in general, we need to use `where` statements for anything at the person/household level to ensure vectorization. `if` statements can only be used for policy parameter values